web/src/narrow.js

```diff
@@ -578,6 +579,9 @@ export function activate(raw_operators, opts) {
         }
         Sentry.getCurrentHub().popScope();
     }
+    if (activity.user_filter) {
+        activity.build_user_sidebar();
+    }
```

Comment:
This should be done in `pm_list.handle_narrow_activated`, I think. Similarly for the `deactivate` code path.

Reply:
Ah yes that makes sense, thanks!